book/opmanual_duckietown/13_traffic_lights_assembly.md

```diff
@@ -480,18 +490,11 @@ See rc-control
 
 -->
 
-### Launch Traffic Lights {#dt-ops-tl-launch status=draft}
-
-SSH into the traffic light, and source environment
-
-    duckiebot $ cd ~/duckietown
-    duckiebot $ git checkout TL-megacity
-
-After this step, launch traffic light node.
+### Launch Traffic Lights {#dt-ops-tl-launch status=ready}
+To download and run the trafficlight docker image, simply run:
 
-    duckiebot $ make traffic-light
+    docker -H ![hostname].local run -dit --privileged --name trafficlight --network=host duckietown/traffic-lights
```

Comment:
there should be a master18 or master19 tag

Reply:
we add that

Reply:
done